qiskit_aer/backends/aer_compiler.py

```diff
@@ -337,12 +337,16 @@ def compile_circuit(circuits, basis_gates=None, optypes=None):
 
 
 def generate_aer_config(
-    circuits: List[QuantumCircuit], backend_options: Options, **run_options
+    circuits: List[QuantumCircuit],
+    default_options: Options,
+    backend_options: Options,
+    **run_options,
```

Comment:
Strictly this is a breaking API change in a public method.  You could avoid the worst of the breaks by doing it as
```python
def generate_aer_config(circuits, backend_options, default_options = None, **run_options):
    default_options = default_options if default_options is not None else {}
    ...
```

Reply:
Thanks. Though this method is an internal method, setting default value looks reasonable.

Reply:
As written in https://github.com/Qiskit/qiskit-aer/pull/1747#discussion_r1141580819, `default_options ` is constant and should not be given as an argument.